docs/package.json

```diff
@@ -72,9 +72,14 @@
   "dependencies": {
     "@docfy/core": "^0.5.0",
     "@docfy/ember": "^0.5.0",
-    "ember-aria-utilities": "*",
+    "ember-aria-utilities": "workspace:*",
     "ember-functions-as-helper-polyfill": "^2.1.1",
     "ember-resources": "^4.8.0",
     "faker": "^5.5.3"
+  },
+  "dependenciesMeta": {
+    "ember-aria-utilities": {
+      "injected": true
+    }
```

Comment:
I'm not familiar with this - care to explain?

Reply:
yeah, I like to think of this as _what should be the default_, but "treat this (injected) package as if it were installed from npm, rather then using links" -- the benefits here are that peers resolve correctly, and the package's devDependencies don't mislead the dependency resolution